product-service/handlers/getProductById.js

```diff
@@ -0,0 +1,24 @@
+import { ProductsService } from '../service/products-service';
+import { RESPONSE } from '../utils/response';
+
+const getProductById = async (event) => {
+  try {
+    const { id } = event.pathParameters || {};
+
+    if (!id || isNaN(id)) {
+      return RESPONSE._400('Product id is not valid');
+    }
+
+    const product = await ProductsService.findById(+id);
+
+    if (!product) {
+      return RESPONSE._404(`Product with id ${id} not found`);
+    }
+
+    return RESPONSE._200(product);
+  } catch (e) {
+    return RESPONSE._500();
+  }
+};
+
+export const handler = async (event) => await getProductById(event);
```

Comment:
Why you didn't use `getProductById` for export?
```
export const getProductById = async (event) => { ... }
```

Reply:
Hi Andrey, 
As far as I remember I had some issue with serverless.yml file and my function wasn't executed. I'll recheck once again, thanks